### ISSUE TITLE
TENSORFLOW MODEL TRAINING

### ISSUE THREAD

**Issue by Doruk-Dilmen** [MaximIntegratedAI/ai8x-training#240]
Hello,

I am working on tensorflow kws20. I need to ask something about training. According to the README.md I have completed the installation. But in the training part, it downloads the dataset and then does not train it. 

![image](https://github.com/MaximIntegratedAI/ai8x-training/assets/91119666/8c82a6e1-5ffa-4805-a1e2-5701559dae30)
![image](https://github.com/MaximIntegratedAI/ai8x-training/assets/91119666/7c5d4951-c983-45ea-a6c2-a258a140b5f5)
![image](https://github.com/MaximIntegratedAI/ai8x-training/assets/91119666/1906fbdb-d8d1-4738-9d4a-927748c20cff)

Why could it be? Am I missing something about installation?

**Comment by aniktash** [comment]
Generally speaking, our Tensorflow branch is outdated and no longer updated or maintained. We are actively supporting Pytorch. So it is highly recommended to move to Pytorch.

In your case, the dataset creation did not go through in the 2nd image. You should see all 35 labels processed. The problem is the librosa update does not like the librosa.effects.time_stretch to have the rate as a positional argument. 
you can update ai8x-training/TensorFlow/datasets/kws20.py, line 294 as:
audio2 = librosa.effects.time_stretch(audio, **rate=rate**)
